### PR TITLE
Todo bug

### PR DESCRIPTION
**Pull request by jack-herrmann**
#13 
I have left in the ToDos for the tests, because they still need to be written.
I could also do that as well though!

### CODE REVIEW

**Comment by daniel-zullo-frequenz** [comment]
The PR title and description may benefit from some additional clarity and could be amended.

Kindly take a moment to read the [commit message guidelines](https://github.com/frequenz-io/rnd/blob/master/doc/coding-standards/commit-messages.md#basic-commit-message-style), particularly focusing on the basic commit message style.

And I understand this is a follow-up PR so probably makes more sense that Luca or Marenz continue with the review as they already checked the initial PR and requested these changes

File: src/frequenz/lib/__init__.py [hunk 1]
```diff
@@ -1,13 +1,10 @@
 # License: MIT
 # Copyright © 2023 Frequenz Energy-as-a-Service GmbH
 
-"""checker to ensure correct datetime usage with aware objects in datetime module.
+"""checker to ensure correct datetime usage with aware objects in datetime module."""
```
Comment (llucax) [inline src/frequenz/lib/__init__.py:4]: While you are at it, maybe capitalize it?

 ```suggestion
"""Checker to ensure correct datetime usage with aware objects in datetime module."""
```

**Comment by llucax** [comment]
Ideally, this should be rebased on top of #16, so we do all the clean ups before adding more code/features.

**Comment by llucax** [comment]
Superseded by #16.

**Pull request closed without merging**